spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/list.html

```diff
@@ -111,15 +66,7 @@
                     }
                 });
 
-
-
-
             });
-
-            function deleteItem(item, consumer) {
-                $("#message").html("Are you sure you want to delete the API key of <strong> " + consumer + "</strong> ? Then " + consumer + " cannot use your API any more");
```

Comment:
This information is lost by unifying the pages. But it could easily be differentiated between the two cases, e.g. if parameter "consumer" is set, display an extended message.